plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -312,6 +312,7 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
          ("eos-vm-oc-enable", bpo::bool_switch(), "Enable EOS VM OC tier-up runtime")
 #endif
          ("enable-account-queries", bpo::value<bool>()->default_value(false), "enable queries to find accounts by various metadata.")
+         ("max-nonprivileged-inline-action-size", bpo::value<uint32_t>()->default_value(4*1024), "maximum allowed size (in bytes) of an inline action for a nonprivileged account")
```

Comment:
Can this directly use `default_max_nonprivileged_inline_action_size` ?

Reply:
can we also support a totem value like `-1` that effectively disables this without having to write out `4294967296` just to be sure that its as high as the consensus limit?

Reply:
It certainly can use the default variable.  Done.

To support -1, it would need to be signed, which would make it unlike literally every other limit.  I can do that, but are you sure you want it to be a beautiful and unique snowflake?